src/Controls/src/Core/Platform/Windows/VisualElementTracker.cs

```diff
@@ -25,7 +25,7 @@
 
 namespace Microsoft.Maui.Controls.Platform
 {
-	// TODO MAUI: can we convert this over to using IView
+	[Obsolete]
```

Comment:
Probably need at least "as of version x" info (and what to use instead) in the obsolete message.

Reply:
👍